tests/integration/oidc/src/test/java/io/helidon/tests/integration/oidc/IntegrationTest.java

```diff
@@ -0,0 +1,133 @@
+package io.helidon.tests.integration.oidc;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Form;
+import javax.ws.rs.core.Response;
+
+import io.helidon.common.http.Http;
+import io.helidon.config.Config;
+import io.helidon.jersey.connector.HelidonProperties;
+import io.helidon.microprofile.server.Server;
+
+import dasniko.testcontainers.keycloak.KeycloakContainer;
+import org.glassfish.jersey.client.ClientConfig;
+import org.glassfish.jersey.client.ClientProperties;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+
+@Testcontainers
+public class IntegrationTest {
```

Comment:
The naming convention should be `...IT` for integration tests, where the name is a usual test name (such as `OidcMultitenantIT`)